1_inventory/src/get_wqp_inventory.R

```diff
@@ -63,7 +63,7 @@ inventory_wqp <- function(grid, char_names, wqp_args = NULL, max_tries = 3){
 
 #' Function to transform WQP sites to use a consistent coordinate reference system
 #'
-#' @param sites data frame containing sites returned from the WQP query; contains
+#' @param sites data frame containing sites returned from the WQP query; must contain
 #' columns lon, lat, and HorizontalCoordinateReferenceSystemDatumName.
 #' @param crs_out character string indicating the desired output coordinate reference
 #' system (crs); options include "NAD83" or "WGS84", defaults to "WGS84"
```

Comment:
The method used to create the doc is slightly inconsistent across functions. See this doc that I've highlighted and also check out [this](https://github.com/lekoenig/ds-pipelines-targets-example-wqp/blob/9252f2c7b779e572c1be5ccdacc507782079972d/1_inventory/src/get_wqp_inventory.R#L1-L19) for a comparison.

I don't have a strong opinion on which way is best overall. the main things that I think are necessary are definitely present in both version (e.g., all arguments listed with brief descriptions and info on data type).

Reply:
Are you referring to the fact that `@description` or `@value` is not included in all functions? If so, I agree that I wasn't too strict in how I set up the documentation. Rather, I primarily focused on making sure the `@param` fields were complete, and then added other fields as I saw fit. Do you think I should attempt to make the documentation 'structure' more consistent?

Reply:
I've tried to clean up the documentation to use a consistent style across all functions.  Let me know if you have any further suggestions for improving the doc style/readability.